ika2btc/processor.go

```diff
@@ -0,0 +1,132 @@
+package ika2btc
```

Comment:
If this package doesn't interact with `ika` then let's rename it - it shouldn't have `ika` in the prefix.